src/SFA.DAS.AANHub.Api/Controllers/CalendarEventsController.cs

```diff
@@ -8,11 +12,28 @@ namespace SFA.DAS.AANHub.Api.Controllers;
 public class CalendarEventsController : ActionResponseControllerBase
 {
     private readonly ILogger<CalendarEventsController> _logger;
+    private readonly IMediator _mediator;
 
     public override string ControllerName => "CalendarEvents";
+    public const string XRequestedByUserHeader = "X-RequestedByUser";
```

Comment:
I found an existing constant [here](https://github.com/SkillsFundingAgency/das-aan-hub-api/blob/a4d4e4f5b02f6c986d95a447bb221b9cbef66880/src/SFA.DAS.AANHub.Api/Common/Constants.cs#L7)  Just change the value to match yours and also change the class name from `PostRequestHeaders` to `RequestHeaders`

Reply:
Done, please could you re-review?